livre/Eq/Config.cpp

```diff
@@ -224,6 +233,22 @@ void Config::resetCamera( )
 
 bool Config::init()
 {
+#ifdef LIVRE_USE_RESTCONNECTOR
+    if( _impl->framedata.getNetworkParameters()->useRESTConnector )
+    {
+        std::string publisherSchema = _impl->framedata.getNetworkParameters()->zeqSchema
+                                      + PUBLISHER_SCHEMA_SUFFIX;
+
+        _impl->_rcPublisherPtr.reset( new zeq::Publisher( lunchbox::URI( publisherSchema ) ) );
+        _impl->_restConnectorPtr.reset( new restconnector::RestConnector(
+                                            _impl->framedata.getNetworkParameters()->hostName,
+                                            _impl->framedata.getNetworkParameters()->port ) );
+        _impl->_restConnectorPtr->run(  _impl->framedata.getNetworkParameters()->zeqSchema );
+    }
+#else
+    _impl->_rcPublisherPtr.reset( new zeq::Publisher( lunchbox::URI( "restresp://" ) ) );
```

Comment:
We want to be able to use zeq without RESTConnector. For that we should have a default schema.